frontend/.eleventy.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Eleventy config function.
+ *
+ * Typing idea borrowed from https://github.com/11ty/eleventy/discussions/2089
+ *  @param {import("@11ty/eleventy/src/UserConfig")} eleventyConfig
+ *  @returns {ReturnType<import("@11ty/eleventy/src/defaultConfig")>}
+ */
+
+const pluginNavigation = require('@11ty/eleventy-navigation');
+
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addWatchTarget('tailwind.config.js');
+  eleventyConfig.addPassthroughCopy('src/fonts');
+  eleventyConfig.addPassthroughCopy('src/images');
+  eleventyConfig.addPassthroughCopy('src/**/*.js');
+  eleventyConfig.addPassthroughCopy('src/**/*.css');
+  eleventyConfig.addPassthroughCopy('src/*.webmanifest');
```

Comment:
I guess we don't need these lines because these assets are referenced using tilde.

Reply:
these are removed.